doc/ref_model/chapters/chapter08.md

```diff
@@ -94,6 +94,59 @@ Figure 8-3 shows a conceptual Telco Operator Platform Architecture. The Cloud In
 
 The Cloud Services and the Cloud Resources Brokers provide value-added services in addition to the fundamental capabilities like service and resource discovery.  These Brokers are critical for a multi-cloud environment to function and utilise cloud specific plugins to perform the necessary activities. These Brokers can, for example, provision and manage environments with resources and services for Machine Learning (ML) services, Augmented/Virtual Reality, or specific industries.
 
+<a name="8.2.4"></a>
+### 8.2.4 Multi-Cloud Interactions Model
+<a name="8.2.4.1"></a>
+### 8.2.4.1 Introduction
+To realise a federated cloud requires definition and agreement of set of APIs. These APIs should allow each of the parties to interact cooperatively and need to cover the management layer: business management and service operations interactions; as well as the data plane, customer and user, transactions and conversational interfaces.
```

Comment:
Could a single plane of glass (a broker) utilise the underlying cloud's APIs but with centralised monitoring and alerting so as to be able to take global-level actions particularly in situations with failures, load, resource constraints, etc.

Reply:
@pgoyal01 

As per comment above on CaaS:
This needs refinement and thought, particularly around how Fault and Performance Management should be managed in these scenarios, which I have allowed for under: Manage Resource - Monitor Resources.

I would like your hear your thoughts on Broker and "single plane of glass".

In typical use "Broker" is external party acting as intermediary between the other parties.

This was the role that OASIS was initially aiming to support with its set of standards (including TOSCA), but the pattern has not really taken off in provision of Cloud Services, to a great extend. The main models being direct consumption and reseller.

In the top figure: "Figure 8-3 - Conceptual Architecture of a Telco Operator Platform" which had the "Cloud Service Broker" and the "Cloud Resource Broker" I always saw these as Orchestration functions, bringing together individual "sets of virtualised cloud resources" (or VIMs of both OpenStack and Kubernetes variety) and playing role of the Brokers outlined.

I think your question, could be answered with a yes, but this is a choice based on the extent to which the consumer wishes to control everything centrally vs delegate the control away.

Could I specify the desired desired behavior globally, but get the manage of this to be automatically distributed down to the various delegated layers ?

Point for discussion.

Reply:
@pgoyal01 @walterkozlowski  I have updated Multi-Cloud Interaction Model to include two different Brokerage models:
- API Brokerage
- Cloud Brokerage , which provide "Single Pane of Glass".